docs/reference/dashboard-monitoring-with-elastic-agent.md

```diff
@@ -86,6 +86,7 @@ Check out [Installing {{agent}}](docs-content://reference/fleet/install-elastic-
 6. Configure the integration to collect metrics.
 
     * Make sure that **Metrics (Elastic Agent)** is turned on (default), and **Metrics (Stack Monitoring)** is turned off.
+   * For the best experience with the {{ls}} dashboards, we recommend collecting all of the metrics. Turning off metrics will result in incomplete or missing visualizations.
```

Comment:
Can we add a tip tag for these? not sure about the rendering in a bulleted list, but I think seeing this called out a little more might help.

```suggestion
    ::::{tip}
     For the best experience with the {{ls}} dashboards, we recommend collecting all of the metrics. Turning off metrics will result in incomplete or missing visualizations.
       ::::

```

Reply:
Hm., if we make it with tips, I think it makes sense to move before (* Make sure that...) instructions.
I will see with other changes how it looks like and make proper tip layout.

Reply:
WDYT this change? - https://github.com/elastic/logstash/pull/17479/commits/6eb7fe6603c32a4b73fd9db8f70153f472082394